Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -68,6 +93,15 @@ final class ManageViewPledgeViewController: UIViewController {
     _ = self.closeButton
       |> \.accessibilityLabel %~ { _ in Strings.Dismiss() }
       |> \.width .~ Styles.minTouchSize.width
+
+    _ = self
```

Comment:
The title should be handled by the view model, because it depends on the project state (live vs finished)

Reply:
Apologies, completely missed this